### ISSUE TITLE
DB/Quests: Great-father Winter is Here! (ID 7021 & 7024 (Horde)) - quest_ender in Ironforge

### ISSUE THREAD

**Issue by illfated** [TrinityCore/TrinityCore#23988]
**Description:**

2 of the 3 Horde Winter Veil seasonal quests named "Great-father Winter is Here!" can not be completed by horde players, because the current quest ender is NPC 13444 (Alliance) Great-father Winter in Ironforge(!).

- https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?quest=7021/great-father-winter-is-here (Thunder Bluff -> Ironforge)
- https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?npc=13431/whulwert-copperpinch (Thunder Bluff)
- https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?quest=7024/great-father-winter-is-here (Undercity -> Ironforge )
- https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?npc=13429/nardstrum-copperpinch (Undercity)
- https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?npc=13444/greatfather-winter (Ironforge)
- https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?npc=13445/greatfather-winter (Orgrimmar)

**Current behaviour:**

When picking up the quest "Great-father Winter is Here!" in either Thunder Bluff or Undercity, the quest can not be delivered to Great-father Winter in Orgrimmar, because these 2 quests use the Alliance Ironforge (dwarf) NPC (13444) as its quest_ender instead of the expected orc Great-father Winter NPC (13445) in Orgrimmar.

**Expected behaviour:**

The orc NPC Great-father Winter (13445) in Orgrimmar needs to be the quest_ender for the quests 7021 & 7024 (Great-father Winter is Here!), because to my knowledge, it has never been possible for horde players to hand in quests to hostile NPCs (unless they are scripted to become friendly for quest hand-ins and also not reside in any of the main cities of a player faction).

**TL;DR** : quest ID 7021 & 7024 needs to have their quest_ender NPC changed from 13444 to 13445 in order to be completable.

**Steps to reproduce the problem:**

1. `.go creature id 13431` (Thunder Bluff) or `.go creature id 13429` (Undercity)
2. Pick up the quest "Great-father Winter is Here!" from the NPC.
3. Try handing in the quest to Great-father Winter, NPC 13445 in Orgrimmar.
4. Notice that the NPC does not have any ❔ above his head and that he only has text gossip for you.

**Branch(es):**

3.3.5 -- I don't know the status of these quests on master.

**TC rev. hash/commit:** 

TrinityCore rev. 26faca5 2019-12-19 22:27:31 +0100 (3.3.5 branch) (Unix, RelWithDebInfo, Static)
```
Using SSL version: OpenSSL 1.1.1c  28 May 2019 (library: OpenSSL 1.1.1c  28 May 2019)
Using Boost version: 1.67.0
Using MySQL version: 8.0.18
Using CMake version: 3.13.4
Compiled on: Linux 5.3.0-1008-azure
Automatic database updates are enabled for the following databases: Auth, Characters, World
Worldserver listening connections on port 8085
Realmlist (Realm Id: 1) configured in port 8085
VMAPs status: Enabled. LineOfSight: 1, getHeight: 1, indoorCheck: 1
MMAPs status: Enabled
maps directory located in ./maps. Total size: 251987687 bytes
vmaps directory located in ./vmaps. Total size: 658130721 bytes
mmaps directory located in ./mmaps. Total size: 1997886944 bytes
Using enUS DBC Locale as default. All available DBC locales: enUS 
Using World DB: TDB 335.19121
```

**Operating system:** Unix (https://tcubuntu.northeurope.cloudapp.azure.com/)

**Note:**
I don't currently have a working local DB or core setup (due to unresolved issues with Visual Studio), so I am unable to verify the TDB contents for these quests and the NPCs involved. Otherwise, I would have posted a suggested solution.

**Comment by Aokromes** [comment]
I suppose:

```sql
UPDATE `creature_questender` SET `id` = 13445 WHERE `quest` IN (7021,7024);

**Comment by illfated** [comment]
Thanks a lot. I was too uncertain to make a suggestion myself.
